src/ec/packages/ec-component-page-header-standardised/page-header-standardised.story.js

```diff
@@ -9,17 +11,46 @@ import demoMetaTitleDescriptionContent from './demo/data--meta-title-description
 import pageHeaderStandardised from './ecl-page-header-standardised.html.twig';
 import notes from './README.md';
 
-demoTitleContent.breadcrumb.icon_file_path = defaultSprite;
-demoMetaTitleContent.breadcrumb.icon_file_path = defaultSprite;
-demoMetaTitleDescriptionContent.breadcrumb.icon_file_path = defaultSprite;
+// Labels for the groups.
+const requiredGroupId = 'Mandatory elements';
+const optionalGroupId = 'Optional elements';
+
+const preparePageHeaderStandardised = data => {
+  data.breadcrumb.icon_file_path = defaultSprite;
+  data.title = text('title', data.title, requiredGroupId);
+  data.breadcrumb = object('breadcrumb', data.breadcrumb, requiredGroupId);
+
+  if (data.meta) {
+    data.meta = text('meta', data.meta, optionalGroupId);
+  }
+  if (data.description) {
+    data.description = text('description', data.description, optionalGroupId);
+  }
+
+  data.extra_classes = text(
+    'extra_classes (comma separated)',
+    '',
+    optionalGroupId
+  );
```

Comment:
If i use this knob the html is update with a comma in between classes. The solution is to separate by space or to use the array knob

Reply:
ah true, @Joosthe , but what is wrong here is just that label saying (comma separated), because it is not what we expect for extra classes, they have been always defined as a single string, so yes, "space" separated but that's not even worth to be mentioned. I'll change this everywhere ;)